Gopkg.lock

```diff
@@ -1,6 +1,12 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  name = "github.com/davecgh/go-spew"
```

Comment:
`go-spew` and `go-difflib` is for debugging?  They don't seem to be used below.

Reply:
they are used by assert